kebechet/managers/version/version.py

```diff
@@ -203,6 +203,18 @@ def run(  # type: ignore
 
         trigger: BaseTrigger
 
+        if (
+                self.parsed_payload
+                and utils._is_merge_event(self.parsed_payload)
+                and utils._is_release_version_pr(self.parsed_payload)
+        ):
+            tag_version =  utils._get_version(self.parsed_payload)
+            with cloned_repo(self) as repo:
+                _LOGGER.info(f"Creating Tag of version {tag_version}")
+                _LOGGER.info(repo.git.execute(["git","tag", f"{tag_version}", f"{utils._get_merge_commit_sha(self.parsed_payload)}"]))
+                _LOGGER.info(repo.git.execute(["git","push", "origin", f"{tag_version}"]))
```

Comment:
I think simple push to server would be problematic for us, 
as the branch name is same as tag name.
lets use explicit push of tag.
```suggestion
                _LOGGER.info(repo.git.execute(["git", "push", "origin", f"refs/tags/v{version}"]))
```